### ISSUE TITLE
Expand curse range to 200ish

### ISSUE THREAD

**Issue by Tarr-OHOL** [jasonrohrer/OneLife#496]
The curse range by no means should be infinite like it has been in the past but due to mechanics such as a bloody corpse the only way to have a town curse someone trolling is either:

Curse them and allow them to run around causing chaos.

Kill them then proceed to have the town find the corpse and then curse. 

With the expanded death stagger someone can get about 200 tiles away from town while wounded which then makes cursing a game of hide and seek. Also, someone murdered will show up as inhospitable _______ if in a bad biome meaning it's impossible to get the name.

**Comment by jasonrohrer** [comment]
So what's the suggestion here?

Cursing by name at a larger radius?

But part of your report is that it's hard to find someone's name if they run away wounded, right?

**Comment by TwistedHBG** [comment]
This is related to this issue so I'm not opening a new one, but I wanted to ask what's the time limit for cursing? Tarr said on the forums that it's 5 minutes after death, and I found a commit from April 25th (https://github.com/jasonrohrer/OneLife/commit/b359d3362c8ace336b836a0e02c4938b81554c58) that confirms that to be the case. Looking at the code it also seems to say it's 5 minutes.

But I distinctly remember cursing someone after death on stream two weeks ago and I went and checked the timestamps, they die at 03:12:20 and I curse them at 03:20:50, a full 8 minutes after death.

Either the curse time limit isn't working as intended, or my curse token got consumed and the curse whiffed. Stream link here for context: <https://www.twitch.tv/videos/532804788?t=3h12m24s>
